apis/clusters/v1beta1/opensearch_types.go

```diff
@@ -283,6 +283,10 @@ func (oss *OpenSearchSpec) areDCsEqual(b []*OpenSearchDataCentre) bool {
 	}
 
 	for i := range b {
+		// TODO remove nodes number and node size from OpenSearch data centre
+		b[i].DataCentre.NodesNumber = a[i].DataCentre.NodesNumber
+		b[i].DataCentre.NodeSize = a[i].DataCentre.NodeSize
```

Comment:
Do you want to compare these fields or what?

Reply:
This fields should be removed for opensearch. I need them to be equal to pass check

Reply:
Removed this check since fields are removed